sdk/core/Azure.Core/tests/HttpPipelineRequestContentTests.cs

```diff
@@ -196,5 +197,55 @@ public void CamelCaseContent()
 
             CollectionAssert.AreEqual(expected.ToArray(), destination.ToArray());
         }
+
+        [Test]
```

Comment:
Can you add test cases that also validate if the model only implements IPersistableModel and not IJsonModel?

Can you add test cases that also validate a mixed model that can be read/written in both xml and json formats and both formats should work?